arc-mlir/src/include/arc/Ops.td

```diff
@@ -56,7 +77,7 @@ def MakeVector
   let verifier = [{ return customVerify(); }];
 }
 
-def ArcKeep : Arc_Op<"keep", []> {
```

Comment:
Don't drop the Arc prefix, I think the namespace for these operations is flat, so we want our own unique prefix.

Reply:
Is there a rule for deciding which we want to have `Arc` as a prefix for?

Reply:
Looking at the other dialects, it looks like all tablegen defs has a prefix in order to not clash when a tablegen file is included.

Reply:
True, maybe it is best to prefix everything with Arc as well?

Reply:
I think so